test/arcade/potts/env/location/PottsLocationTest.java

```diff
@@ -836,12 +824,16 @@ public void splitVoxels_YZPlaneDirectionRandomZero_updatesLists() {
         voxelsASplit.add(new Voxel(0, 0, 0));
         voxelsASplit.add(new Voxel(0, 1, 1));
         voxelsASplit.add(new Voxel(0, 2, 1));
+        
         voxelsBSplit.add(new Voxel(1, 0, 2));
         voxelsBSplit.add(new Voxel(1, 1, 1));
         voxelsBSplit.add(new Voxel(2, 2, 2));
         voxelsBSplit.add(new Voxel(2, 2, 0));
+
+        Voxel center = loc.getCenter();
+        Plane yzPlane = new Plane(new Int3D(center.x, center.y, center.z), Direction.YZ_PLANE.getVector());
```

Comment:
I would add a `Plane` constructor like:

```java
Plane(Voxel point, Direction direction)
```

to reduce some of the repeated code.